src/MenuButton/MenuButtonItem.tsx

```diff
@@ -1,13 +1,26 @@
 import React from "react";
 import PropTypes from "prop-types";
-import iconSelection from "src/icons/selection.json";
+import iconSelection from "../icons/selection.json";
 import Row from "../Row";
+import { IconName } from "../types/Icon.types";
+
+interface MenuButtonItemProps {
+  /** Display label for menu item */
+  label: string;
+  /** Selection handler */
+  onSelect: () => void;
```

Comment:
`onSelect` not used?

edit: oh I see, it's for the parent to pull from the props?  perhaps a future refactor could group things with this handle

```ts
export interface SelectableChildProps {
   /** Selection handler */
  onSelect: () => void;
}

interface MenuButtonItemProps extends SelectableChildProps {
   ...
}
```